src/routes/campaign/[slug]/+page.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+    import { onMount } from 'svelte';
+    export let data;
+
+    let value = 'loading...';
+
+    // if you weant to show the page instantly
+    // then update it with the fetched data
+    // then use this approach:
+    onMount(() => {
+        setTimeout(() => {
+            value = `fetched data for slug:  ${data.slug}`
+        }, 1000)
+    })
+</script>
+
+{#if value}
+<div>Here is data:</div>
+<div>{JSON.stringify(value)}</div>
+{:else}
+<div>Waiting on data</div>
```

Comment:
For you to peruse and play with: https://loading.io/css/

Reply:
Good candidate for the eventual ui theme settings

Reply:
My favorite of the front page ones is the ellipses one.  `source`